src/TypeDefinition.js

```diff
@@ -305,6 +305,7 @@ export type NavigationDrawerScreenOptions = NavigationScreenOptions & {
     | ((
       options: { tintColor: ?string, focused: boolean },
     ) => ?React.Element<*>),
+  drawerLockMode?: 'unlocked' | 'locked-close' | 'locked-open', // defaults `unlocked`
```

Comment:
maybe `locked-closed`?

Reply:
@MSSPL-PiyaleeMaiti 
No its not working now, this PR needs to be merged onto master